### PR TITLE
Bump Go to 1.13.8

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

Use latest version available of Go.

### Motivation

We want to benefit from latest improvements.

### Additional Notes

We've noticed that the new builders are not generating exactly the same
`tracer-ebpf.o` output. That may be due to a slight package change in clang
(8.0.1-7 -> 8.0.1-8 even if a bit unlikely as it is a package change).
We've validated that the tests are still passing properly with the new binary and
we're then updating it.

Link to public builders change: DataDog/datadog-agent-buildimages#30

 - [x] Must not be merged for `7.18.0`, we want to ship this with `7.19.0`.
 - [x] Still needs to update the builder tags in `gitlab.yaml`
 - [x] CircleCI image has to be pushed

### CODE REVIEW

**Review by olivielpeau — DISMISSED** [review]
Thanks, looks good to merge once CI is green, the gitlab runner tags are updated and a full pipeline passes 👍

**Review by hush-hush — CHANGES_REQUESTED** [review]
Should we not also update the gitlab builders ?

**Review by hush-hush — DISMISSED** [review]

File: releasenotes/notes/go-1-13-8-d98857855401db29.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+other:
+  - |
+    The Agent, Logs Agent and the system-probe are now compiled with Go ``1.13.8``
```
Comment (olivielpeau) [inline releasenotes/notes/go-1-13-8-d98857855401db29.yaml]: Also APM and process agents no?

(in that case, I guess writing `all the Agent binaries` makes sense)

**Review by olivielpeau — APPROVED** [review]
LGTM once the gitlab pipeline is ✅

Just one question on the release note.

What's the QA plan for this? (we can follow up on the card)

**Review by albertvaka — APPROVED** [review]

**Review by p-lambert — APPROVED** [review]

File: releasenotes/notes/go-1-13-8-d98857855401db29.yaml (hunk 1 above)
Comment (remeh) [reply releasenotes/notes/go-1-13-8-d98857855401db29.yaml]: Got a bit too much excited by seeing all those ✅
Let me send another quick PR for that...

**Pull request merged**